packages/web/pages/_app.tsx

```diff
@@ -17,6 +18,7 @@ const app: React.FC<AppProps> = ({ pageProps, Component }) => {
       </Head>
       <Web3ContextProvider>
         {!pageProps.hidePageHeader && <PageHeader />}
+        {!pageProps.hidePageHeader && <MobileFooter />}
```

Comment:
isn't this a bit confusing? hidePageHeader would hide the Footer

Reply:
They both have the same functionality.

Reply:
I do agree it could be confusing from variable.
Should discuss whether or not it's appropriate though!

Reply:
@mikeki, I've changed this to `hideAppDrawer` btw! :)